src/NoSingleLineRecords.elm

```diff
@@ -0,0 +1,103 @@
+module NoSingleLineRecords exposing (rule)
+
+{-|
+
+@docs rule
+
+-}
+
+import Elm.Syntax.Declaration as Declaration exposing (Declaration)
+import Elm.Syntax.Node as Node exposing (Node(..))
+import Elm.Syntax.Range as Range exposing (Range)
+import Elm.Syntax.TypeAnnotation as TypeAnnotation exposing (TypeAnnotation)
+import Review.Fix as Fix exposing (Fix)
+import Review.Rule as Rule exposing (Rule)
+
+
+{-| Reports record definitions in type aliases that are written on a single line.
+
+    config =
+        [ NoSingleLineRecords.rule
+        ]
+
+
+## Fail
+
+    type alias Person =
+        { name : String, age : Int, address : Address }
+
+
+## Success
+
+    type alias Person =
+        { name : String
+        , age : Int
+        , address : Address
+        }
+
+
+## When (not) to enable this rule
+
+You don't ever need to enable this rule. This is a code style that can easily be reviewed manually as appropriate.
+
+
+## Try it out
+
+You can try this rule out by running the following command:
+
+```bash
+elm-review --template sparksp/elm-review-rules-to-avoid/example --rules NoSingleLineRecords
```

Comment:
If you don't intend to publish this rule, you likely need to link to `/preview` rather than `/example`

Reply:
Ah, good catch! That's just boilerplate but will update that and try to remember for the others too.